exporter/elasticsearchexporter/internal/objmodel/objmodel.go

```diff
@@ -175,6 +176,14 @@ func (doc *Document) AddAttribute(key string, attribute pcommon.Value) {
 	}
 }
 
+// AddEvents converts and adds span events to the document.
+func (doc *Document) AddEvents(key string, events ptrace.SpanEventSlice) {
+	for i := 0; i < events.Len(); i++ {
+		e := events.At(i)
+		doc.AddAttributes(flattenKey(key, e.Name()), e.Attributes())
```

Comment:
What about the event timestamp? That's pretty important information

Reply:
aha, Indeed, I will add `time_unix_nano` into attributes.